src/server/utilities/canAdvertise/index.js

```diff
@@ -0,0 +1,13 @@
+const canAdvertise = headers => {
+  const isAdvertiseCombine = headers['x-ip_is_advertise_combined'];
+  const isUkCombined = headers['x-ip_is_uk_combined'];
```

Comment:
This looks great, thanks! I've just taken another look at the page `/display/geoip/GeoIP+Headers` in confluence, which tells us all about the different GeoIP headers and how they are calculated. After our example mapping, we said we'd follow GNL in checking both `'x-ip_is_advertise_combined'` and `'x-ip_is_uk_combined'` - however according to that confluence page, `'x-ip_is_advertise_combined'` already takes into account the value of `'x-ip_is_uk_combined'`. Can we try this without having to check `'x-ip_is_uk_combined'`, and test that we don't render ads in the UK when we only check `'x-ip_is_advertise_combined'`? This will reduce the complexity of this logic, and mean that we have to vary on one fewer header when we're cacheing upstream :D

Reply:
I've removed the `'x-ip_is_uk_combined'` header and everything appears to work as before. Thanks